### ISSUE TITLE
librexgen no longer does case.

### ISSUE THREAD

**Issue by jfoug** [magnumripper/JohnTheRipper#2107]
https://github.com/teeshop/rexgen/commit/9468680a1d86e25f6e08d98fbcd32980c2ec6d8b

We still have this in code (within our command line arg), but the flag that gets set never is used any more (after Jan's update last december:  https://github.com/magnumripper/JohnTheRipper/commit/f3cd0e6393278eba7e8456666c1b27a8df8e4938#commitcomment-16945258)

We need to find out if there are plans to ever bring that function back, and if not, then remove this option from the regex interface.

Note, the =alpha:case does work (after I renamed case to cased).

**Comment by magnumripper** [comment]
Is this still an issue? Or was it re-implemented by eg. https://github.com/teeshop/rexgen/commit/9e4e557 ?

**Comment by jfoug** [comment]
No, this is still an issue.  casing 'can' be done, but it is not done with a simple function call like before.  I think you have to incorporate it into the regex expression, but that makes for some differences in runtime.

**Comment by magnumripper** [comment]
> Note, the =alpha:cased does work

How? The below doesn't.
```
$ ../run/john --stdout -regex='alpha:cased=\0' -stdin <<< magnum
Using default input encoding: UTF-8
magnum
1p 0:00:00:00 0.00% 100.0p/s magnum
```
Am I doing it wrong? I'm probably confused.

**Comment by magnumripper** [comment]
doc/README.librexgen is full of typos and incorrect (as of now) info. It should be updated.

For example, apparently you need to write `-regex='alpha:cased=[regex]` whereas the manual says `-regex='alpha=cased=[regex]`. You can say `-regex:'alpha:cased=[regex]` but the second one must be a `:` and the third one must be a `=`. Not sure if we should change the docs or the code.

Typos are things like "rexex" and "stand along"

**Comment by solardiz** [comment]
We don't even know whether we'll keep this functionality longer-term, so we certainly don't intend to spend time on it before our upcoming release.

**Comment by solardiz** [comment]
Should we perhaps remove all mentions of `case` from `doc/README.librexgen` and then close this issue?
